central/reports/config/datastore/datastore_impl_postgres_test.go

```diff
@@ -120,6 +131,38 @@ func (s *ReportConfigurationPostgresDatastoreTests) TestMultipleReportNotifiers(
 	s.Equal(reportConfig, foundReportConfig)
 }
 
+func (s *ReportConfigurationPostgresDatastoreTests) TestDeleteCollectionWithReportConfig() {
```

Comment:
Add another test that creates a reportConfig with collection id set to empty. Ensure that it works.